app/src/main/java/agency/five/codebase/android/movieapp/ui/component/FavoriteButton.kt

```diff
@@ -0,0 +1,53 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.ui.theme.Blue
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Icon
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun FavoriteButton(
+    isSelected: Boolean,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        contentAlignment = Alignment.Center,
+        modifier = modifier
+    ) {
+        Box(
+            contentAlignment = Alignment.Center,
+            modifier = Modifier
+                .size(50.dp)
```

Comment:
You removed fixed size from outer `Box`, but still have fixed size here and on `Icon`. If you now try to pass a modifier to `FavoriteButton` with different size (for example if you pass `Modifier.size(70.dp)`), this will look broken. You can fix that by setting `.fillMaxSize()` here instead of hardcoding size, and on `Icon` you can set `.fillMaxSize().padding(8.dp)` and center it. That way your component will scale properly.

Reply:
Removed all fixed sizes.